src/pages/bulkImport/utils/useBulkImportFields.js

```diff
@@ -185,13 +210,42 @@ export default function useBulkImportFields() {
     },
   ];
 
+  const allAvailableFields = [
+    ...additionalFlatfileFields,
+    ...flatfileEncounterFields,
+    ...flatfileSightingFields,
+  ];
+  const allLabels = map(
+    allAvailableFields,
+    field => field?.label,
+    [],
+  );
+  const duplicates = filter(
+    allAvailableFields,
+    (field, idx) => allLabels.indexOf(field?.label) !== idx,
+  );
+  let disambiguatedAvailableFields;
+  duplicates.forEach(duplicate => {
```

Comment:
I figured that this forEach was ok, given that duplicates should be small and only a .map is called each time therein.